ui/src/app/utils/strings.ts

```diff
@@ -15,5 +15,8 @@ interface extractStringByDelimiterByPosArguments{
 }
 
 export const extractStringByDelimiterByPos = ({str,delimiter=".",occurence=-1}:extractStringByDelimiterByPosArguments) => {
+  if (str=="domain.com")
+    return str
+  else
   return str.split(delimiter).slice(occurence).join(delimiter)
```

Comment:
@nixonakash01 There is no need for else here

Reply:
For strings like Java.lang.String and Java,lang.List, the delimiter is needed to slice the string to String or List

Reply:
if delimiter is not present, then it will display the whole string like Java.lang.String or Java.lang.List

Reply:
I think you misunderstood. I am not asking you to remove line no 21 here. If if condition succeeds it will return the str. If it doesn't, `str.split(delimiter).slice(occurence).join(delimiter)` will be returned. I dont see a need for else

Reply:
oh. sorry i misunderstood. will commit.